src/OrchardCore.Modules/OrchardCore.Flows/Assets/js/flows.edit.js

```diff
@@ -57,4 +57,10 @@ $(function () {
         // default - change inner html with value
         $(this).html(val);
     });
+
+    $(document).on('click', '.btn.toggleAll', function (e) {
+        e.preventDefault();
```

Comment:
Should fix the issue with double clicks.

Reply:
I thought that - i guess it should not just toggle instead it should expand all. or collapse all.

What is your opinion on scope of the button? the closest flowpart or all content card within a document?

Reply:
Fixed, currently it will collapse/expand all within the document